modules/platform/nuxeo-platform-document-routing/nuxeo-routing-core/src/main/java/org/nuxeo/ecm/platform/routing/core/impl/DocumentRoutingServiceImpl.java

```diff
@@ -1161,6 +1163,26 @@ public void reassignTask(CoreSession s, final String taskId, final List<String>
         });
     }
 
+    protected Map<String, Serializable> getWorkflowContextualInfo(CoreSession session, Task item) {
+        String workflowInstanceId = item.getProcessId();
+        String nodeId = item.getVariable(DocumentRoutingConstants.TASK_NODE_ID_KEY);
+        Map<String, Serializable> eventInfo = new HashMap<>();
+
+        if (session != null) {
+            GraphNode node = null;
+            if (StringUtils.isNotBlank(workflowInstanceId)) {
```

Comment:
Please use `isNotEmpty` instead. This is a technical value and there's no reason for it to contain extra spaces.